### DISCUSSION TITLE
CommandPost on Windows

### DISCUSSION THREAD

**Discussion by romansafiullin** [CommandPost/CommandPost#3317] [Technical Support] [answered]
Someone in the Adobe Premiere Pro Editors Facebook group gave CommandPost a shoutout for its use with Speed Editor. That would be a dream come true for me. However, I work on a PC. Are there any plans to bring CommandPost to Windows? If not, what steps would be needed to facilitate this conversion? Thanks.

**Comment by latenitefilms** [comment, marked as answer]
Unfortunately not. It’s almost impossible for us to port CommandPost to Windows as it’s basically designed for Mac technologies. Sorry!

I have seen other developers working on trying to add Speed Editor support for Premiere on Windows though, so I’m sure someone will offer this at some point.
